cszczepaniak/cribbly-backend
# Get rid of all mocking in tests

# What I added (link any issues addressed)
I added fake repositories for all of our data and then consumed them in all of our tests. This enabled us to completely get rid of Moq.

I also added/refactored a few tests here and there.

# How to test it
Run `dotnet test` and see that all tests are passing; CI should also be passing.
